### PR TITLE
Added forbes.com

### PR DESCRIPTION
**Pull request by kovaka**
This commit adds ability to bypass paywalls on forbes.com.

### CODE REVIEW

File: manifest-ff.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "*://*.ed.nl/*",
     "*://*.elpais.com/*",
     "*://*.examiner.com.au/*",
+    "*://*.forbes.com/*",
```
Comment (kimkha) [inline manifest-ff.json:25]: This should be after firstthings.com

Comment (iamadamdev) [reply manifest-ff.json:25]: Actually I don't care about order in the manifest.
We can start adding new sites to the bottom of the list.

File: README.md [hunk 2]
```diff
@@ -181,4 +182,4 @@ Usually premium articles cannot be bypassed as they are behind a hard paywall.
 * PRs are welcome.
 
 ### Show your support
-* I am not asking for donations or anything like that, all I ask is that you star this repo.
\ No newline at end of file
+* I am not asking for donations or anything like that, all I ask is that you star this repo.
```
Comment (iamadamdev) [inline README.md:185]: What's this change for?

Comment (kimkha) [reply README.md:185]: Some editors (IntelliJ, or even github) automatically add new line break at the end of file.

**Comment by iamadamdev** [comment]
Fix merge conflicts please

**Comment by kimkha** [comment]
As mention in #584 , there is no paywall on forbes.com (I don't use adblock). So I guess this PR could be closed.

**Pull request closed without merging**